x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_table.tsx

```diff
@@ -60,7 +60,7 @@ const exceptionReferenceModalInitialState: ReferenceModalState = {
 
 export const ExceptionListsTable = React.memo(() => {
   const { formatUrl } = useFormatUrl(SecurityPageName.rules);
-  const [{ loading: userInfoLoading, canUserCRUD }] = useUserData();
+  const [{ loading: userInfoLoading, canUserCRUD, canUserREAD }] = useUserData();
```

Comment:
I think you could change to useUserInfo instead since you dont need to dispatch any actions. Long term we want to make that switch.

const { loading: userInfoLoading, canUserCRUD, canUserREAD } = useUserInfo();

and then change the test to reflect it

Reply:
Thanks @jamster10 ! If it's ok, I'll look to implement that separately. Since it's a bug fix for this next BC, trying to minimize changes to the code as much as I can while still fixing the bug.